base/setup/lib/utils/osdetect.c

```diff
@@ -320,7 +320,7 @@ CheckForValidPEAndVendor(
 
             RtlStringCbCopyNW(VendorName->Buffer, VendorName->MaximumLength,
                               pvData, BufLen * sizeof(WCHAR));
-            VendorName->Length = wcslen(VendorName->Buffer) * sizeof(WCHAR);
+            VendorName->Length = (USHORT)wcslen(VendorName->Buffer) * sizeof(WCHAR);
```

Comment:
We don't have `RtlUnicodeStringCbCopyStringN`, do we ?

Reply:
What would that do?

Reply:
> We don't have `RtlUnicodeStringCbCopyStringN`, do we ?

No we don't.

Reply:
> What would that do?

Copy a string into a UNICODE_STRING, as name suggests :slightly_smiling_face: 
https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/ntstrsafe/nf-ntstrsafe-rtlunicodestringcbcopystringn